docker/spec/dependabot/docker/file_parser_spec.rb

```diff
@@ -259,197 +259,11 @@
           it "has the right details" do
             expect(dependency).to be_a(Dependabot::Dependency)
             expect(dependency.name).to eq("ubuntu")
-            expect(dependency.version).to eq("12.04.5")
+            expect(dependency.version).to eq("18305429afa14ea462f810146ba44d4363ae76e4c8dfc38288cf73aa07485005")
             expect(dependency.requirements).to eq(expected_requirements)
           end
         end
 
-        context "for a private registry" do
```

Comment:
Were these private registry tests moved elsewhere or no longer necessary?

Reply:
I removed them because I think they are no longer necessary since the code they validated was removed.

Previously, when an image reference in a Dockerfile included only a digest, we would need to access the registry to figure out the corresponding version for that digest. Now we set the digest as the version directly and don't try to look up a numeric equivalent for the digest.

Basically, we now use exclusively the information provided in the Dockerfile for parsing dependencies, so we no longer need any registry access here.